data-raw/configuration.R

```diff
@@ -1,30 +1,28 @@
-# TODO list
-# * Only pull data once per species and then combine the pull results with the
-#   configuration matrix again
-# * Fix how format_data returns an object without the nwfscSurvey class instead
-#   it is class(data) > [1] "tbl_df" "tbl" "data.frame"
-# * for vessel_year, might want a different level scaling things might not have
-#   to give this to grid
+# Read in the configuration file within data-raw that define what model set-up
+# to apply by species
 configuration <- tibble::as_tibble(read.csv(
   file.path("data-raw", "configuration.csv")
 ))
 
-data <- configuration %>%
+# Download the data and filter the data based upon species-specific
+# depths and latitudes in the configuration file
+data <- configuration |>
   # Row by row ... do stuff then ungroup
-  dplyr::rowwise() %>%
```

Comment:
I did not go through the package and update all of them since I was trying to keep the changes limited to the topic of this pull request.  I can post an issue to get this updated throughout so we can tackle that in a future pull request.